Website/Src/Startup.cs

```diff
@@ -59,6 +59,16 @@ public void Configure(IApplicationBuilder app)
                     defaults: new { controller = "Home", action = "Beta" });
             });
 
+            app.UseSwagger(c =>
+            {
+                c.PreSerializeFilters.Add((swagger, httpReq) => swagger.Host = httpReq.Host.Value);
```

Comment:
Can you add a comment for what this line does? For me at least it's not very clear.

Reply:
Sure, this was mainly copy / paste.  Let me see what all I can remove.  Also looking into build time generation of this.